FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -68,5 +67,58 @@ public void ExecuteTestInvalidCredentials()
             FireboltException exception = Assert.Throws<FireboltException>(() => conn.Open());
             Assert.IsTrue(exception.Message.Contains("403") || exception.Message.Contains("429"));
         }
+
+        [Test]
+        public void ExecuteSelectTimestampNtz()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10 23:01:02.0'::timestampntz");
+            DateTime dt = new DateTime(2022, 5, 10, 23, 1, 2, 0);
+            NewMeta newMeta = ResponseUtilities.getFirstRow(command.Response);
+            Assert.That(newMeta.Meta, Is.EqualTo("TimestampNtz"));
+            Assert.That(newMeta.Data[0], Is.EqualTo(dt));
+        }
+
+        [Test]
+        public void ExecuteSelectTimestampPgDate()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10'::pgdate");
+            DateTime dt = new DateTime(2022, 5, 10, 0, 0, 0);
+            NewMeta newMeta = ResponseUtilities.getFirstRow(command.Response);
+            Assert.That(newMeta.Meta, Is.EqualTo("Date"));
+            Assert.That(newMeta.Data[0], Is.EqualTo(DateOnly.FromDateTime(dt)));
+        }
+
+        [Test]
+        public void ExecuteSelectTimestampTz()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10 23:01:02.0 Europe/Berlin'::timestamptz");
+
+            DateTime dt = DateTime.Parse("2022-05-10 21:01:02Z");
```

Comment:
I thought the time difference between Berlin and UTC is 1 hour? Why is there 2 hours here? Can we set local timezone so this is not platform-dependent?

Reply:
2022-05-10 21:01:02.0+00 is what is returned by the server. Since I'm creating the time with the timezone (Z), it's actually not platform-dependent (time is not converted to the platform timezone and remains in Zulu time)

Reply:
I asked postgres and it's expected
![image](https://user-images.githubusercontent.com/10419887/212296281-9fd7d1a3-8992-46b4-b8f0-2c6895c76990.png)

Reply:
I just double checked but it's actually correct according to timezoneconverter.com 😅

Reply:
For an example I used it gave me a 1-hour offset.
![image](https://user-images.githubusercontent.com/93913847/212300068-f52bcab5-0c11-40dd-aa27-c1862a28c53c.png)
I understand now, it's daylight saving time that's affecting it. Setting the full date in converter also gives 2 hour offset.